### PR TITLE
Added javax.activation to resolve class not found error.

### PR DESCRIPTION
**Pull request by sabariish**
Was able to start the server only after adding the below dependency:
               <dependency>
			<groupId>javax.activation</groupId>
			<artifactId>activation</artifactId>
			<version>1.1.1</version>
		</dependency>
Please review.

### CODE REVIEW

**Comment by eranharel** [comment]
@sabariish this is weird. `javax.activation` should come with the Java SDK
Can you please provide the output of `java -version`?

```
$ java -version
openjdk version "1.8.0_242"
OpenJDK Runtime Environment (build 1.8.0_242-8u242-b08-0ubuntu3~18.04-b08)
OpenJDK 64-Bit Server VM (build 25.242-b08, mixed mode)
```

**Comment by sabariish** [comment]
java version "12.0.1" 2019-04-16
Java(TM) SE Runtime Environment (build 12.0.1+12)
Java HotSpot(TM) 64-Bit Server VM (build 12.0.1+12, mixed mode, sharing)

File: pom.xml [hunk 1]
```diff
@@ -1,200 +1,208 @@
...
+			<version>1.9.5</version>
+		</dependency>
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
```
Comment (eranharel) [inline pom.xml]: I verified your change.
Please reformat the file with the original indentation (2 spaces for a tab), and I will merge

Comment (sabariish) [reply pom.xml]: Done. Please review.
Thanks

**Pull request merged**